dipy/viz/regtools.py

```diff
@@ -384,6 +389,9 @@ def overlay_slices(L, R, slice_index=None, slice_type=1, ltitle='Left',
     colorImage[..., 0] = ll * (ll > ll[0, 0])
     colorImage[..., 1] = rr * (rr > rr[0, 0])
 
+    if ret_slice:
```

Comment:
I would rather use this function to return slice always and define a new function with a slice image as input and create a figure with an overlay